test/groups/test_heisenberg_group.jl

```diff
@@ -0,0 +1,49 @@
+using LieGroups, Random, Test
+
+using ManifoldsBase: ℂ
+
+s = joinpath(@__DIR__, "..", "LieGroupsTestSuite.jl")
+!(s in LOAD_PATH) && (push!(LOAD_PATH, s))
+using LieGroupsTestSuite
+
+@testset "Heisenberg group" begin
+    G = HeisenbergGroup(1)
+    g1, g2, g3 = [1.0 2.0 3.0; 0.0 1.0 -1.0; 0.0 0.0 1.0],
+    [1.0 4.0 -3.0; 0.0 1.0 3.0; 0.0 0.0 1.0],
+    [1.0 -2.0 1.0; 0.0 1.0 1.1; 0.0 0.0 1.0]
+
+    X1, X2, X3 = [0.0 2.0 3.0; 0.0 0.0 -1.0; 0.0 0.0 0.0],
+    [0.0 4.0 -3.0; 0.0 0.0 3.0; 0.0 0.0 0.0],
+    [0.0 -2.0 1.0; 0.0 0.0 1.1; 0.0 0.0 0.0]
+
+    properties = Dict(
+        :Name => "Heisenberg group",
+        :Points => [g1, g2, g3],
+        :Vectors => [X1, X2, X3],
+        :Rng => Random.MersenneTwister(),
+        :Functions => [
+            compose,
```

Comment:
```suggestion
            adjoint,
            compose,
```

also here, my tutorial will also cover tests and which functions to usually test ;)